src/search/documents/user.py

```diff
@@ -90,11 +95,27 @@ def prepare_full_name_suggest(self, instance):
 
     def prepare_full_name(self, instance):
         try:
-            return f"{instance.author_profile.first_name} {instance.author_profile.last_name}"
+            return (
+                f"{instance.author_profile.first_name} "
+                f"{instance.author_profile.last_name}"
+            )
         except Exception:
             # Some legacy users don't have an author profile
             return f"{instance.first_name} {instance.last_name}"
 
     def prepare_is_verified(self, instance):
         """Prepare the is_verified field for Elasticsearch indexing"""
         return instance.is_verified
+
+    def prepare_is_suspended(self, instance):
+        """Prepare the is_suspended field for Elasticsearch indexing"""
+        if instance is None:
+            return None
```

Comment:
This check isn't necessary since the OpenSearch DSL framework will only invoke `prepare_*` methods during the indexing of actual model instances.